ui/src/app/components/buttons.tsx

```diff
@@ -275,7 +275,13 @@ export const Clickable = ({
   return (
     <Interactive
       as={as}
-      {...childProps}
+      {...{
```

Comment:
When Clickables were disabled, they had not been showing the right cursor. 

https://github.com/all-of-us/workbench/assets/24514630/c88206bd-d32a-48a5-95cf-c0969b435b32